src/pages/memberhsip/components/termsOfService/TermsOfService.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
```

Comment:
I think this component (and related files) could be in it's own directory, e.g. `src/pages/termsOfService`

Reply:
Changed.